libs/perun/dialogs/src/lib/member-tree-view-dialog/member-tree-view-dialog.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="group-theme">
+  <h1 mat-dialog-title>{{'SHARED_LIB.PERUN.COMPONENTS.MEMBER_TREEVIEW_DIALOG.TITLE' | translate}}</h1>
+  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <div *ngIf="!loading" mat-dialog-content class="mr-0">
+    <p>{{'SHARED_LIB.PERUN.COMPONENTS.MEMBER_TREEVIEW_DIALOG.INFO' | translate}}</p>
+
+    <mat-form-field class="filter-field ml-2">
+      <input matInput [formControl]="formControl"
+             placeholder="{{'SHARED_LIB.PERUN.COMPONENTS.MEMBER_TREEVIEW_DIALOG.FILTER' | translate}}">
+    </mat-form-field>
```

Comment:
The filter field itself is small, I would make it bigger - or reuse the filter field we normally use